x/sequencer/genesis.go

```
@@ -10,6 +10,15 @@ import (
 // state.
 func InitGenesis(ctx sdk.Context, k keeper.Keeper, genState types.GenesisState) {
 	k.SetParams(ctx, genState.Params)
+	// Validate all other genesis fields are empty
+	genesisFields := []int{
```

can we just do 
`if len(genState.SequencerList) != 0 {
        return fmt.Errorf("invalid genesis state: SequencerList should be empty")
   }`

---

it's in the init genesis, which doesn't except error to be retuned, but panicing on failure.

---

Then we can panic instead of returning error maybe

---

ha you mean that genesisFields is not needed because it's one field. gotcha.